tests/integration_tests/tests/usage_collector_base.py

```diff
@@ -49,9 +47,21 @@ def run_collector_scripts_and_assert(self, messages):
                                join(LOG_PATH, LOG_FILE))
 
     def clean_timestamps(self):
-        # this is necessary for forcing the collector scripts to actually run
+        # This is necessary for forcing the collector scripts to actually run
         # in subsequent tests, despite not enough time passing since last run
         run_postgresql_command(
             self.env.container_id,
-            "UPDATE usage_collector SET hourly_timestamp=0, daily_timestamp=0 "
-            "WHERE id=0")
+            "UPDATE usage_collector SET hourly_timestamp=NULL, "
+            "daily_timestamp=NULL")
+
+    def clean_usage_collector_log(self):
+        # We need to clean the usage_collector log before each test, because
+        # each test uses it for asserting different values.
+        old_usage_log = join(LOG_PATH, self._testMethodName)
```

Comment:
so you went with this after all :P

Reply:
Yep, found out that it works with pytest :)